learn-stan/articles.qmd

```diff
@@ -74,33 +75,39 @@ toc-location: left
 
 ## Bayesian Workflow
 
-* Workflow paper
+* Andrew Gelman, Aki Vehtari, Daniel Simpson, Charles C. Margossian, Bob Carpenter, Yuling Yao, Lauren Kennedy, Jonah Gabry, Paul-Christian Bürkner, Martin Modrák. 2020. [Bayesian Workflow](https://arxiv.org/pdf/2011.01808)
 
 * Aki papers - PSIS-LOO-CV                       
 
                        
 ## Books using Stan
```

Comment:
Should we add Active Statistics and Regression and Other Stories?

Reply:
added.